third_party/parser/cc/driver.cc

```diff
@@ -16,7 +16,7 @@ base_driver::base_driver(ruby_version version, const std::string& source, const
 }
 
 typedruby25::typedruby25(const std::string& source, const struct builder& builder)
```

Comment:
Should we also rename this class? My understanding is that we're aiming to a kind of version-agnostic parser with Sorbet.

Reply:
feel free! it's probably just an oversight.